templates/devfiles/gradle/devfile.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2021 IBM Corporation and others
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+schemaVersion: 2.0.0
+metadata:
+  name: java-openliberty-gradle
+  version: 0.6.0
+  description: Java application Gradle built stack using the Open Liberty runtime
+  alpha.build-dockerfile: "{{.OUTERLOOP_DOCKERFILE_GRADLE_LOC}}"
+  alpha.deployment-manifest: "{{.DEVFILE_DEPLOY_YAML_GRADLE_LOC}}"
+starterProjects:
+  - name: user-app
+    git:
+      remotes:
+        origin: 'https://github.com/OpenLiberty/application-stack-starters.git'
+components:
+  - name: devruntime
+    container:
+      image: {{.STACK_IMAGE_GRADLE}}
+      memoryLimit: 1512Mi
```

Comment:
Where does this 1512 number come from ?   I guess just our Maven devfile?   Wonder where, not that you have to answer.

Reply:
I would prefer if we did not specify anything and pick whatever the default is which if i am not mistaken was unlimited last time i took a look at this. However, to be consistent with the original value set for the maven counterpart, i used the same value. We should pick up this discussion outside of this PR and decide what to do going forward.

Reply:
OK..opened https://github.com/OpenLiberty/application-stack/issues/155